File: The Core/removeArrayPart.cs
int[] removeArrayPart(int[] inputArray, int l, int r) {
    List<int> output = new List<int>();
    
    for(int i = 0; i < Math.Min(l, r); i++){
        output.Add(inputArray[i]);
    }
    
    for(int i = Math.Max(l,r) + 1; i < inputArray.Length; i++){
        output.Add(inputArray[i]);
    }
    
    return output.ToArray();
}

File: The Core/List Forest Edge/39. Remove Array Part/removeArrayPart.py
def removeArrayPart(inputArray, l, r):
    output = list()

    for x in range(0, min(l, r)):
        output.append(inputArray[x])
        
    for x in range(max(l, r) + 1, len(inputArray)):
        output.append(inputArray[x])
    
    return output
